src/include/rrdb/rrdb.code.definition.h

```diff
@@ -18,5 +18,6 @@ DEFINE_STORAGE_READ_RPC_CODE(RPC_RRDB_RRDB_TTL)
 DEFINE_STORAGE_READ_RPC_CODE(RPC_RRDB_RRDB_GET_SCANNER)
 DEFINE_STORAGE_READ_RPC_CODE(RPC_RRDB_RRDB_SCAN)
 DEFINE_STORAGE_READ_RPC_CODE(RPC_RRDB_RRDB_CLEAR_SCANNER)
+DEFINE_STORAGE_READ_RPC_CODE(RPC_DETECT_HOTKEY)
```

Comment:
`DEFINE_STORAGE_READ_RPC_CODE` will define a rpc code whose action is read data from storage, start or stop detecting hotkey is not this rpc, it should not be defined here.